pkg/healthcheck/healthcheck.go

```diff
@@ -264,6 +264,13 @@ func (hc *HealthChecker) allCategories() []category {
 						return hc.kubeAPI.CheckVersion(hc.kubeVersion)
 					},
 				},
+				{
+					description: "minimum kubectl version is installed",
```

Comment:
can you match the wording in the previous check (and then update linkerd/website#194 accordingly)?
```
is running the minimum kubectl version
```

Reply:
Done